### PR TITLE
http_uri supports binary strings in OTP 20

### PR DESCRIPTION
**Pull request by tsloughter**

### CODE REVIEW

**Comment by deadtrickster** [comment]
Hi, cool! Any idea why that crazy fails on Travis?

**Comment by yurrriq** [comment]
Looks like elvis broke.

**Comment by tsloughter** [comment]
Huh... I don't know what the hell is going on in travis. Tests ran fine for me locally.

**Comment by yurrriq** [comment]
For now can you comment out the eunit prehook?

https://github.com/elli-lib/elli/blob/develop/rebar.config#L27

**Comment by yurrriq** [comment]
Might as well comment out rebar_lint too but I'm mobile so fetching line numbers is dumb.

**Comment by yurrriq** [comment]
https://github.com/elli-lib/elli/blob/develop/rebar.config#L24

File: rebar.config [hunk 1]
```diff
@@ -21,10 +21,10 @@
 
 {project_plugins, [
   {coveralls, "1.3.0"},
```
Comment (yurrriq) [inline rebar.config]: trailing comma 😢

**Review by yurrriq — CHANGES_REQUESTED** [review]

**Comment by yurrriq** [comment]
Thanks!

**Pull request merged**